hope_horizon/backend/views/group_user_view.py

```diff
@@ -0,0 +1,40 @@
+from rest_framework import viewsets, status
+from rest_framework.response import Response
+from backend.serializers import GroupUserSerializer
+from backend.models import CustomGroup, GroupUser
+from rest_framework.permissions import IsAuthenticated
+from django.db.models import Q
+
+class GroupUserViewSet(viewsets.ModelViewSet):
+    permission_classes = [IsAuthenticated]
+    serializer_class_group_user = GroupUserSerializer
+
+    def list(self, request):
+        user = request.user
+        id = request.query_params.get("group_id", None)
+        try:
+            group = CustomGroup.objects.get(pk=id)
+        except CustomGroup.DoesNotExist:
+            return Response(status=status.HTTP_404_NOT_FOUND)
+        if not group.is_active:
+            return Response(status=status.HTTP_404_NOT_FOUND)
+        try:
+            group_user = GroupUser.objects.filter(group=group).filter(user=user).get()
+            if not group_user.is_owner:
+                return Response(status=status.HTTP_403_FORBIDDEN)
+        except GroupUser.DoesNotExist:
```

Comment:
also test this case 

you can use: coverage run manage.py test backend and coverage report -m to get coverage report